File: mRemoteNG/Connection/Connection.Protocol.SSH2.vb
Namespace Connection
    Namespace Protocol
        Public Class SSH2
            Inherits Connection.Protocol.PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.ssh
                Me.PuttySSHVersion = Putty_SSHVersion.ssh2
            End Sub

            Public Enum Defaults
                Port = 22
            End Enum
        End Class
    End Namespace
End Namespace

File: mRemoteV1.ConvertedToC#/Connection/Connection.Protocol.SSH2.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
namespace mRemoteNG.Connection
{
	namespace Protocol
	{
		public class SSH2 : Connection.Protocol.PuttyBase
		{

			public SSH2()
			{
				this.PuttyProtocol = Putty_Protocol.ssh;
				this.PuttySSHVersion = Putty_SSHVersion.ssh2;
			}

			public enum Defaults
			{
				Port = 22
			}
		}
	}
}
